src/Kernel/client/plotting.ts

```diff
@@ -1,12 +1,219 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-function importScript(url: string) {
-    let script = document.createElement("script");
-    script.setAttribute("src", url);
-    document.body.appendChild(script);
+// This is a bit of a hack needed to map the requireJS
+// call made by TypeScript onto the URL that Jupyter
+// makes our kernelspec available at.
+//
+// Using this hack, we can split the type import and
+// the runtime import apart, then glue them back
+// together using a declare global to solve TS2686.
+/// <amd-dependency path="/kernelspecs/iqsharp/chart.js" name="Chart" />
+import type * as ChartJs from "chart.js";
+declare global {
+    const Chart: typeof ChartJs;
 }
 
+export interface Complex {
+    Real: number;
+    Imag: number;
+    Magnitude: number;
+    Phase: number;
+};
+
+export interface DisplayableState {
+    n_qubits: number;
+    div_id: string;
```

Comment:
Not sure what the style guide is on this team, but in most cases, JavaScript uses CamelCase.

Reply:
This is an odd one. Our style guide is generally to use `camelCase`, as you note, but this interface constrains the type of a Jupyter message. Since Jupyter uses `snake_case` for its JSON serialization schemas, this interface uses `snake_case` to match as an intentional exception to the typical style guide.

Reply:
Ah gotcha, that makes sense. Thanks!